ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/SystematicAnalysisEngine.java

```diff
@@ -73,16 +77,32 @@ boolean isFallback() {
      * Throw a SensitivityComputationException if the computation fails.
      */
     void run(LinearRaoData linearRaoData) {
+        SystematicSensitivityAnalysisResult sensiResults = runSensitivityAnalysis(linearRaoData);
+
+        Map<String, Double> loopflows = new HashMap<>();
+        if (this.runLoopflow) {
+            loopflows = computeLoopflowAndCheckLoopflowConstraint(linearRaoData);
+        }
+
+        setResults(linearRaoData, sensiResults, loopflows);
+    }
```

Comment:
I don't understand why you added this part to this PR. Isn't it part of the EPIC dedicated to the loopflows?

Reply:
Its a resolution of the conflicts with the loopflow PR.